### DISCUSSION TITLE
SuiteAdvanced Global Search

### DISCUSSION THREAD

**Discussion by Travis048** [burkybang/SuiteAdvanced-Feedback#85] [Q&A] [answered]
Ever since the last NetSuite update, I no longer have the dropdown menu on the left side of the global search. Is this no longer a feature of SuiteAdvanced or is there a setting I am missing? I used this feature often and would appreciate any info. Thanks!

**Comment by burkybang** [comment]
I haven't figured out how to recreate that for NetSuite 2023.1 yet. The new global search input field is weird.

I wasn't sure how much that feature would be missed. You asking about it lets me know at least _you_ were using it!

I'll give it another go.

**Comment by Travis048** [comment]
Thank you! Just getting such a quick response is awesome! I definitely do
use it. It is way easier than having to type the prefixes all the time.

On Fri, Apr 28, 2023 at 4:32 PM Adam Smith ***@***.***> wrote:

> I haven't figured out how to recreate that for NetSuite 2023.1 yet. The
> new global search input field is weird.
>
> I wasn't sure how much that feature would be missed. You asking about it
> lets me know at least *you* were using it!
>
> I'll give it another go.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/85#discussioncomment-5758266>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7PYVNVTOUMPKR2ZWCO6EALXDQSNBANCNFSM6AAAAAAXPPNFTM>
> .
> You are receiving this because you authored the thread.Message ID:
> <burkybang/SuiteAdvanced-Feedback/repo-discussions/85/comments/5758266@
> github.com>
>


-- 
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
--
*Travis Corsi*
*Operations Manager*
*Jendco Safety Supply, Inc**.*

Office: 614-449-9600

Cell: *740-404-7748*

www.jendcosafety.com - ***@***.***

**Reply by burkybang** [reply, marked as answer]
I successfully reimplemented it! It'll be in version 26.8.0 coming out later today.

**Reply by burkybang** [reply]
![image](https://user-images.githubusercontent.com/927886/235376649-bf72c4ff-6212-4c0d-ba8e-52591823b9a9.png)

**Comment by Travis048** [comment]
That's great news Adam! I am glad you were able to figure it out and thanks
for doing it so quickly.

On Sun, Apr 30, 2023 at 5:18 PM Adam Smith ***@***.***> wrote:

> [image: image]
> <https://user-images.githubusercontent.com/927886/235376649-bf72c4ff-6212-4c0d-ba8e-52591823b9a9.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/85#discussioncomment-5767994>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7PYVNTNMGH3ZU32T62KHFTXD3JI5ANCNFSM6AAAAAAXPPNFTM>
> .
> You are receiving this because you authored the thread.Message ID:
> <burkybang/SuiteAdvanced-Feedback/repo-discussions/85/comments/5767994@
> github.com>
>


-- 
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
--
*Travis Corsi*
*Operations Manager*
*Jendco Safety Supply, Inc**.*

Office: 614-449-9600

Cell: *740-404-7748*

www.jendcosafety.com - ***@***.***

**Reply by burkybang** [reply]
No prob! Thanks for the feedback.

Are there any other prefixes you'd like added to the dropdown?

**Comment by Travis048** [comment]
You're welcome!

The only other prefixes that I can think of right now that might be helpful
would be Journals and Inventory Counts

On Mon, May 1, 2023 at 8:32 AM Adam Smith ***@***.***> wrote:

> No prob! Thanks for the feedback.
>
> Are there any other prefixes you'd like added to the dropdown?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/85#discussioncomment-5771282>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7PYVNUORNO57EJRIICQ2I3XD6UPHANCNFSM6AAAAAAXPPNFTM>
> .
> You are receiving this because you authored the thread.Message ID:
> <burkybang/SuiteAdvanced-Feedback/repo-discussions/85/comments/5771282@
> github.com>
>


-- 
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
--
*Travis Corsi*
*Operations Manager*
*Jendco Safety Supply, Inc**.*

Office: 614-449-9600

Cell: *740-404-7748*

www.jendcosafety.com - ***@***.***

**Reply by burkybang** [reply]
Cool. I added the following record types for the next release:

- _Entity_
  - Partner
- _Item_
  - Discount
- _Transaction_
  - Cash Sale
  - Opportunity
  - Journal
  - Inventory Count

**Comment by Travis048** [comment]
Awesome, thanks!

On Wed, May 3, 2023 at 10:45 AM Adam Smith ***@***.***> wrote:

> Closed #85
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/85> as
> resolved.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/burkybang/SuiteAdvanced-Feedback/discussions/85>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7PYVNTLKZJPVNBSKE4G5GTXEJVRNANCNFSM6AAAAAAXPPNFTM>
> .
> You are receiving this because you authored the thread.Message ID:
> <burkybang/SuiteAdvanced-Feedback/repo-discussions/85/discussion_event/645240
> @github.com>
>


-- 
-------------------------------------------------------------------------------------------------------------------------------------------------------------------------
--
*Travis Corsi*
*Operations Manager*
*Jendco Safety Supply, Inc**.*

Office: 614-449-9600

Cell: *740-404-7748*

www.jendcosafety.com - ***@***.***
